Sources/SwiftBuildSupport/SwiftBuildSystem.swift

```diff
@@ -265,7 +266,11 @@ public final class SwiftBuildSystem: SPMBuildCore.BuildSystem {
             )
 
             do {
-                try await withSession(service: service, name: self.buildParameters.pifManifest.pathString, packageManagerResourcesDirectory: self.packageManagerResourcesDirectory) { session, _ in
+                let toolchainPath = self.buildParameters.toolchain.swiftCompilerPath
```

Comment:
I think the toolchain base path should also be part of the Toolchain protocol itself rather than expanded here in SwiftBuildSystem.

Also because on some platforms it's usr/local/bin instead of usr/local, so I'd rather that logic be kept In UserToolchain itself.

Reply:
There's not an obvious way to add that requirement right now because the SwiftPM-level toolchain type conflates toolchains, SDKs, and cross compilation bundles